tests/frameworks/test_gevent_autotrace.py

```diff
@@ -0,0 +1,72 @@
+# (c) Copyright IBM Corp. 2021
+# (c) Copyright Instana Inc. 2020
+
+import importlib
+import os
+import unittest
+import socket
+
+import gevent
+from gevent import monkey
+from instana import apply_gevent_monkey_patch
+
+
+class TestGEventAutoTrace(unittest.TestCase):
+    def setUp(self):
+        # Ensure that the test suite is operational even when Django is installed
+        # but not running or configured
+        os.environ['DJANGO_SETTINGS_MODULE'] = ''
+
+        self.default_patched_modules = ('socket',  'time', 'select',
+                'os', 'select', 'threading', 'os','ssl','subprocess', 'signal',
+                'queue',)
```

Comment:
Just a small formatting glitch that caught my eye.

```suggestion
        self.default_patched_modules = ('socket', 'time', 'select',
                'os', 'select', 'threading', 'os', 'ssl', 'subprocess', 'signal',
                'queue',)
```

Reply:
done